components/eam/src/chemistry/mozart/mo_exp_sol.F90

```diff
@@ -148,6 +184,11 @@ subroutine exp_sol( base_sol, reaction_rates, het_rates, extfrc, delt, xhnm, nco
           end do
        else
           do i = 1,ncol
+             do k = 1,ltrop(i) ! for emissions in the stratosphere: NO2 as high as 15 km
```

Comment:
Why NO2?

Reply:
OK, I see it's explained in the PR description.

Reply:
NO2 is implicit tracer, but its emission (up to 15 km) may over troposphere. I feel these emission should be considered. When it fall into troposphere, it can react with other species.
If you think this shouldn't be taken into consideration, I can change it back.

Reply:
Since NO2 is defined as an implicit tracer, it doesn't handled here in the explicit solver. I will update the comment to make this clearer.